frontend/src/routes/Governance/discovered/ByCluster/DiscoveredByClusterPage.test.tsx

```diff
@@ -54,7 +55,7 @@ describe('DiscoveredByClusterPage', () => {
               name: 'check-policy-reports',
               namespace: 'managed1',
               compliant: 'Compliant',
-              remediationAction: 'inform',
+              responseAction: 'inform',
```

Comment:
Why did you change this?

Reply:
That way the UI code doesn't have to differentiate between `remediationAction` and `enforcementAction` based on the policy type. The grouping.ts file does that for you.